srcalc/src/main/webapp/WEB-INF/views/fragments/dropdownInputs.jsp

```diff
@@ -1 +1,9 @@
-<!-- Dropdowns are not yet supported. Don't load anything onto the page. -->
\ No newline at end of file
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+
+<%-- This file cannot be a jspf extension because it is not included statically. --%>
+<%-- This jsp expects for displayItemParam to be defined. --%>
+<form:select path="${displayItemParam.varPath}">
+    <form:options items="${displayItemParam.options}"/>
+</form:select>
+<%-- There cannot be any errors since the only options given are valid. --%>
```

Comment:
It is still a good practice to include an errors tag. Yes, the UI as designed only allows specification of valid options but technically errors are still possible (via a programming error, malformed browser request, etc.).